apps/web/src/components/ClickableLogo/ClickableLogo.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import './ClickableLogo.scss';
+import Icon, { iconColor } from '../Icon/Icon';
+import Wordmark from '../Wordmark/Wordmark';
+import { ReactComponent as RedditIcon } from '../../assets/redditIcon.svg';
+
+/**
+ * Renders a logo and wordmark component together within a navbar.
+ *
+ * @returns {React.ReactElement} A JSX element as a clickable reddit logo component.
+ */
+
+const ClickableLogo = (): React.ReactElement => {
+  /**
+   * handles click of logo
+   *
+   * @returns {string} confirming the logo has been clicked
+   *
+   * @description This function is called when the user clicks on the logo. It logs a message to the console to confirm that the click has been registered.
+   */
+
+  const handleClick: () => void = () => {
+    console.log('Click Registered');
```

Comment:
Can this be refactored to call a callback that is passed as a prop to make this more flexible? Like what if we want to have it do one thing when clicked when its on the main page but have it do something else when its somewhere else?